lib/cli/src/generators/ANGULAR/angular-helpers.ts

```diff
@@ -50,9 +50,23 @@ export function editStorybookTsConfig(tsconfigPath: string) {
   writeFileAsJson(tsconfigPath, tsConfigJson);
 }
 
-export function isDefaultProjectSet() {
-  const angularJson = readFileAsJson('angular.json', true);
-  return angularJson && !!angularJson.defaultProject;
+export function getDefaultProjectName(angularJson: any): string {
+  const { defaultProject, projects } = angularJson;
+
+  if (projects.storybook) {
+    return 'storybook';
+  }
+
+  if (defaultProject) {
+    return defaultProject;
+  }
+
+  const firstProjectName = Object.keys(projects)[0];
+  if (firstProjectName) {
+    return firstProjectName;
+  }
+
+  throw new Error('No angular projects found');
```

Comment:
I added a method that runs on the beginning of the `npx sb init` process and it will retunrn the following error:
`Could not find a project in your Angular workspace. \nAdd a project and re-run the installation`